CompetitionTemplate/src/competition/Robot.java

```diff
@@ -3,10 +3,26 @@
 
 import competition.operator_interface.OperatorCommandMap;
 import competition.subsystems.SubsystemDefaultCommandMap;
+import competition.subsystems.vision.InertJetsonServer;
+import competition.subsystems.vision.JetsonServer;
+import competition.subsystems.vision.NetworkedJetsonServer;
 import xbot.common.command.BaseRobot;
+import xbot.common.injection.RobotModule;
 
 public class Robot extends BaseRobot {
 
+    public Robot() {
+        super();
```

Comment:
A super constructor won't be invoked unless called explicitly, so I wanted to make sure that all other systems are initialized as well.